### PR TITLE
[NR-17340] [Debian 11] use which for tune2fs/blkid path

### PR DESCRIPTION
**Pull request by ignatenko-personal**

### CODE REVIEW

File: components/os-installer.sh [hunk 1]
```diff
@@ -180,12 +180,15 @@ cp -a /mnt/sbin/fsck.ext2 /tmp/utils/
 cp -a /mnt/sbin/fsck.ext4 /tmp/utils/
 echo "Done."
 
+TUNE2FS=`which tune2fs`
```
Comment (YuriiHordynskyi) [inline components/os-installer.sh:183]: Will it return the path to the onie image's tune2fs utility or one which is installed on the Debian image?
The idea here was to use the utility from the installed Debian image.
```
sh-5.1# find / -name tune2fs
/sbin/tune2fs

```

Comment (konstcranky) [reply components/os-installer.sh:183]: It will search in PATH :)

**Comment by ignatenko-personal** [comment]
The PR has updated according to suggestion of @konstcranky

**Review by konstcranky — APPROVED** [review]

File: components/os-installer.sh [hunk 2]
```diff
@@ -180,12 +180,24 @@ cp -a /mnt/sbin/fsck.ext2 /tmp/utils/
...
+if [[ -z "$TUNE2FS" ]] then;
+    TUNE2FS=/mnt/sbin/tune2fs
+    echo "WARNING: tune2fs isnt found (used fallback /mnt/sbin/tune2fs)"
+fi
+
+BLKID=`which blkid`
+if [[ -z "$BLKID" ]] then;
+    TUNE2FS=/mnt/sbin/blkid
```
Comment (vitalivanov) [inline components/os-installer.sh]: Blkid

**Review by vitalivanov — CHANGES_REQUESTED** [review]

File: components/os-installer.sh (hunk 2 above)
Comment (ignatenko-personal) [reply components/os-installer.sh]: sure(

**Comment by ignatenko-personal** [comment]
> Blkid

fixed

**Review by YuriiHordynskyi — APPROVED** [review]

**Review by vitalivanov — APPROVED** [review]

**Pull request merged**